src/modules/fuzzy-search/look-up.model.ts

```diff
@@ -0,0 +1,18 @@
+import { Column, PrimaryKey, Table, Model } from 'sequelize-typescript';
+import { LookUpAttributes } from './look-up.domain';
+
+@Table({
+  underscored: true,
+  tableName: 'look_up',
+})
+export class LookUpModel extends Model implements LookUpAttributes {
+  @PrimaryKey
+  @Column
+  id: string;
+
+  @Column
+  userUuid: string;
+
+  @Column
+  name: string;
```

Comment:
Reference `File['plainName'] | Folder['plainName']` as the type please

Reply:
Changed on the attributes interface. The model implements it so it's centralized